ElectronicObserver/Window/Tools/SortieRecordViewer/Sortie/Battle/Phase/PhaseTorpedo.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using ElectronicObserver.KancolleApi.Types.Models;
+using ElectronicObserver.Properties.Data;
+using ElectronicObserverTypes;
+using ElectronicObserverTypes.Attacks;
+
+namespace ElectronicObserver.Window.Tools.SortieRecordViewer.Sortie.Battle.Phase;
+
+public enum TorpedoPhase
```

Comment:
maybe extract this into another file

Reply:
Right, forgot about that in the end.
Extracted.